### PR TITLE
Feature/limit versions per collection

### PR DESCRIPTION
**Pull request by jondewijones**
### What

Prevent publishing users adding more than one version to a collection. 

### How to review

- Upload a few versions
- Create a test collection, and add a version (should also add related dataset) to a collection
- Shouldn't let you edit another version or create a new version

### Who can review

Anyone

### CODE REVIEW

File: src/app/views/datasets-new/versions/DatasetVersionsController.jsx [hunk 1]
```diff
@@ -170,7 +181,7 @@ export class DatasetVersionsController extends Component {
                 }    
             });
             versionsList.sort((this.sortByVersionNumber))
-            return versionsList.reverse();
+            return versionsList.reverse();
```
Comment (dandwal) [inline src/app/views/datasets-new/versions/DatasetVersionsController.jsx]: Why are we adding tailing white space?

File: src/app/components/simple-selectable-list/SimpleSelectableListItem.jsx [hunk 2]
```diff
@@ -15,10 +16,15 @@ export default class SimpleSelectableListItem extends Component {
     }
 
     render() {
+        console.log(this.props)
```
[the remark this answers is not in the document]
Comment (jondewijones) [reply src/app/components/simple-selectable-list/SimpleSelectableListItem.jsx]: Fixed. We need a linter to highlight these

File: src/app/views/datasets-new/versions/DatasetVersionsController.jsx (hunk 1 above)
Comment (jondewijones) [reply src/app/views/datasets-new/versions/DatasetVersionsController.jsx]: Fixed. ¯\_(ツ)_/¯

File: src/app/components/simple-selectable-list/SimpleSelectableListItem.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (dandwal) [reply src/app/components/simple-selectable-list/SimpleSelectableListItem.jsx]: Agreed

**Pull request merged**